config/redirects.yml

```diff
@@ -26,6 +26,7 @@ redirects:
   # International content
   "/iqts": "/non-uk-teachers/international-qualified-teacher-status"
   "/funding-and-support/if-you-come-from-outside-england": "/non-uk-teachers/fees-and-funding-for-non-uk-trainees"
+  "/non-uk-teachers/fees-and-funding-for-non-UK-trainees": "/non-uk-teachers/fees-and-funding-for-non-uk-trainees"
```

Comment:
Looks to me like this redirect is already in the redirects file @laurahermionetennant92 

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/4932e43e-1123-48ad-8c5f-ca276ca27e3f)

Reply:
That is above my Github pay grade @gemmadallmandfe! Do I need to take an action?